02-filter/extractCountriesWithFiveCharactersOrFewer.js

```diff
@@ -5,3 +5,12 @@
  */
 
 const COUNTRIES = ["United Kingdom", "Italy", "France", "Portugal", "Greece"];
+
+function extractCountriesWithFiveCharactersOrFewer(countries) {
+    return countries.filter(country => country.length <= 5);
+}
+
+const result = extractCountriesWithFiveCharactersOrFewer(COUNTRIES);
```

Comment:
FIX: remove these lines. It is not necessary to invoke function and log result. You are testing it, aren't you?